manager/raftpicker/raftpicker.go

```diff
@@ -131,3 +140,8 @@ func (c *ConnSelector) Conn() (*grpc.ClientConn, error) {
 	c.cc = cc
 	return c.cc, nil
 }
+
+// Stop cancels tracking loop for raftpicker.
+func (c *ConnSelector) Stop() {
```

Comment:
This must be a blocking stop. Otherwise `ticker.C` could file at the same time as `c.stop` is closed, and we still might call `LeaderAddr` after the raft node has been destroyed.